hook_functions/netif_receive_skb.c

```diff
@@ -0,0 +1,84 @@
+#define TCP_PROTOCOL_CODE 6
```

Comment:
it's already defined in the kernel, right?

Reply:
yep, done